source/blog/2022-01-12-announcing-hanami-200alpha5.html.markdown

```diff
@@ -0,0 +1,206 @@
+---
+title: Announcing Hanami v2.0.0.alpha5
+date: 2022-01-12 11:00:00 UTC
+tags: announcements
+author: Tim Riley
+image: true
+excerpt: >
+  Sensible default logger, full source dirs config, router loading flexibility, route helpers in views
```

Comment:
```suggestion
  Sensible default logger, full source dirs config, faster router loading for code reloading, route helpers in views
```

Reply:
@jodosha Given that we don't have the code reloading side of this sorted out yet, I think I might not apply this suggestion of yours. I reckon it's better for us to tell that story when we have the working code to back it up.